app/components/navigation/google-login-btn.tsx

```diff
@@ -0,0 +1,55 @@
+import Script from "next/script";
+import { useEffect, useState, useRef } from "react";
+
+interface GoogleLogin {
+    credential: string,
+    select_by: string
+}
+
+export default function GoogleLoginButton(){
+    const [scriptLoaded, setScriptLoaded] = useState(false);
+    const handleScriptLoad = () => {
+        setScriptLoaded(true);
+    };
+
+    const googleLoginRef = useRef(null);
+
+    useEffect(()=> {
+        if(scriptLoaded){
+            const google = window.google;
+            google.accounts.id.initialize({
+                client_id: "708154320268-432vdrg2g0h1652frag5vbu8r8qi4ers.apps.googleusercontent.com",
```

Comment:
Is this what needs to be updated to use a `client_id` that is created under the xfactorleaguesite@gmail.com account?

Reply:
Yes, plus in Vercel